packages/repository/src/relations/relation.helpers.ts

```diff
@@ -48,3 +58,85 @@ export async function findByForeignKeys<
 }
 
 type StringKeyOf<T> = Extract<keyof T, string>;
+
+/**
+ * Returns model instances that include related models that have a registered
+ * resolver.
+ *
+ * @param targetRepository - The target repository where the model instances are found
+ * @param entities - An array of entity instances or data
+ * @param filter - Query filter
+ * @param options - Options for the operations
+ */
+
+export async function includeRelatedModels<
+  T extends Entity,
+  Relations extends object = {}
+>(
+  targetRepository: EntityCrudRepository<T, unknown, Relations>,
+  entities: T[],
+  filter?: Filter<T>,
```

Comment:
Should it be `include?: Inclusion,`?

Reply:
This method will be called by [`find`, `findById`](https://github.com/strongloop/loopback-next/pull/3387/files#diff-177253a1afbdd8eb83b99efef86dd415R393), etc. That's why we pass in `filter` instead of `include`.

Reply:
Yes, but this method does not use or honor other properties such as `where`, `fields` from the `Filter`, right? The parameter should be minimally constrained to allow `includeRelatedModels` to work, not the callers.